index.html

```diff
@@ -0,0 +1,177 @@
+            alt="판다마켓 메인 컨텐츠 핫이슈"
+            width="579"
+            height="444"
+          />
+          <div class="main-contents-container">
+            <p class="main-contents-title">Hot item</p>
+            <strong class="main-contents-strong"
+              >인기 상품을<br />확인해 보세요</strong
+            >
+            <p class="main-contents-description">
+              가장 HOT한 중고거래 물품을<br />판다 마켓에서 확인해 보세요
+            </p>
+          </div>
+        </div>
+      </section>
+      <section class="main-section-wrapper">
+        <div class="layout main-contents-wrapper">
+          <div class="main-contents-container left">
+            <p class="main-contents-title">Search</p>
+            <strong class="main-contents-strong"
+              >구매를 원하는 <br />상품을 검색하세요</strong
+            >
+            <p class="main-contents-description">
+              구매하고 싶은 물품은 검색해서 <br />쉽게 찾아보세요
+            </p>
+          </div>
+          <img
+            src="/images/contents-img-02.webp"
+            alt="판다마켓 메인 컨텐츠 서치"
+            width="579"
+            height="444"
+          />
+        </div>
+      </section>
+      <section class="main-section-wrapper">
+        <div class="layout main-contents-wrapper">
+          <img
+            src="/images/contents-img-03.webp"
+            alt="판다마켓 메인 컨텐츠 레지스터"
+            width="579"
+            height="444"
+          />
+          <div class="main-contents-container">
+            <p class="main-contents-title">Register</p>
+            <strong class="main-contents-strong"
+              >판매를 원하는 <br />상품을 등록하세요</strong
+            >
+            <p class="main-contents-description">
+              어떤 물건이든 판매하고 싶은 상품을<br />쉽게 등록하세요
+            </p>
+          </div>
+        </div>
+      </section>
+      <section class="main-bg-section-wrapper">
+        <div class="layout main-bg-contents-wrapper">
+          <div class="main-bg-contents-container">
+            <p class="main-bg-contents-description">
+              믿을 수 있는 <br />판다마켓 중고 거래
+            </p>
+          </div>
+          <img
+            src="/images/footer-img.webp"
+            alt="판다마켓 메인 헤더"
+            width="746"
+            height="397"
+          />
+        </div>
+      </section>
+    </main>
+    <footer class="footer-bg-wrapper">
+      <div class="layout footer-container">
+        <p class="footer-CI">©codeit - 2024</p>
+        <div class="footer-link-container">
+          <a href="/privacy.html" class="footer-link">Privacy Policy</a>
+          <a href="/faq.html" class="footer-link">FAQ</a>
+        </div>
+        <div class="footer-icon-container">
+          <a
+            href="https://www.facebook.com/?locale=ko_KR"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            <img
+              src="/images/facebook-icon.webp"
+              alt="페이스북 아이콘"
+              width="20"
+              height="20"
+            />
+          </a>
```

Comment:
💊 제안  
이 영역은 로고들의 모음이라고 할 수 있습니다. 관련된 요소들끼리 묶인 목록이니 의미록적인 마크업을 작성한다면 list 관련 태그로 작성하시는 것을 추천드립니다.

```suggestion
        <ul class="footer-icon-container">
          <li>
            <a
              href="https://www.facebook.com/?locale=ko_KR"
              target="_blank"
              rel="noopener noreferrer"
            >
              <img
                src="/images/facebook-icon.webp"
                alt="페이스북 아이콘"
                width="20"
                height="20"
              />
            </a>
          </li>
```